AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -361,6 +361,7 @@ class CallingContext: NSObject {
                     }
                 }
             }
+            self.notifyRemoteParticipantIsSpeakingChanged()
```

Comment:
1. I think `notifyRemoteParticipantIsSpeakingChanged` function is completely overlap with `notifyRemoteParticipantIsMutedChanged`, they both notify the view update. Should we combine them into one function like `notifyRemoteParticipantViewChanged`?
2. on the close of `onIsSpeakingChanged`, there already has some view updating business logic. I think we could update the logic to optimize the notification? Something similar to 
 -If the participant is speaking and not in the grid update the view(which's code already exist in the project) 
-otherwise if participant is in the grid, update the view 
-otherwise no update

Reply:
1. True. Initially I thought we could provide optimization on targeted individual updates in the future, but we could also leave the need for separate notifications (if necessary) in the future as well.
2. Good point. We did that with `onIsMutedChanged` as well so it's logical to bring that optimization over.